test/performance/README.md

```diff
@@ -21,3 +23,30 @@ Their name reference the test where they come from, currently `get-*.csv` and `c
 
 -   `.txt` files contain summaries of Locust, the time unit is milliseconds.
 -   `.csv` files contain values from Prometheus, the CPU unit is in milliCPU, the memory unit is bytes.
+
+## Merging results from GitHub Workflows
+
+To merge test results from GitHub Workflows you need to run the following command:
+
+```bash
+export GH_TOKEN="a personal github token"  # Needs read permission for workflows so it can download artifacts
+bash test/performance/merge.sh
+```
+
+A bunch of output will get printed into stdout, but to visualize the results run:
+
+```
+# You need to install `gnuplot` first
+gnuplot -e "titel='Memory during GET'; filename='merge/get-memory.csv'; outfile='get-memory.png'" test/performance/memory.gnuplot
+gnuplot -e "titel='Memory during POST'; filename='merge/create-memory.csv'; outfile='create-memory.png'" test/performance/memory.gnuplot
+gnuplot -e "titel='CPU during GET'; filename='merge/get-cpu.csv'; outfile='get-cpu.png'" test/performance/cpu.gnuplot
+gnuplot -e "titel='CPU during POST'; filename='merge/create-cpu.csv'; outfile='create-cpu.png'" test/performance/cpu.gnuplot
```

Comment:
Do you mean tittle?

```suggestion
gnuplot -e "tittle='Memory during GET'; filename='merge/get-memory.csv'; outfile='get-memory.png'" test/performance/memory.gnuplot
gnuplot -e "tittle='Memory during POST'; filename='merge/create-memory.csv'; outfile='create-memory.png'" test/performance/memory.gnuplot
gnuplot -e "tittle='CPU during GET'; filename='merge/get-cpu.csv'; outfile='get-cpu.png'" test/performance/cpu.gnuplot
gnuplot -e "tittle='CPU during POST'; filename='merge/create-cpu.csv'; outfile='create-cpu.png'" test/performance/cpu.gnuplot
```

Reply:
No, `titel` is the variable. I think it does fly with `title` as a variable because it overrides a builtin function. I don't know if I can call it `name` either.

Reply:
Changed it to `name` so its cleared and do not resemble a typo.